msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2021-05-25 15:06+0000\n"
"Last-Translator: \n"
"Language-Team: Slovenian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.3\n"
"Plural-Forms: nplurals=4; plural=(n%100==1 ? 1 : n%100==2 ? 2 : n%100==3 || n%100==4 ? 3 : 0);\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Project-ID: 289505\n"
"X-Crowdin-Language: sl\n"
"X-Crowdin-File-ID: 670\n"
"Language: sl_SI\n"

#: redbot/cogs/audio/core/events/cog.py:215
msgid "Auto Play started."
msgstr "Samodejno predvajanje se je začelo."

#: redbot/cogs/audio/core/events/dpy.py:40
msgid "You have attempted to run Audio's Lavalink server on an unsupported architecture. Only settings related commands will be available."
msgstr "Poskusili ste zagnati Lavalink strežnik modula Audio na nepodprti arhitekturi. Na voljo bodo samo ukazi, povezani z nastavitvami."

#: redbot/cogs/audio/core/events/dpy.py:70
msgid "I'm missing permissions in this server, Please address this as soon as possible.\n\n"
"Expected Permissions:\n"
msgstr "Nimam nekaterih dovoljenj, prosim odpravi to težavo čim hitreje:\n\n"
"Potrebna dovoljenja:\n"

#: redbot/cogs/audio/core/events/dpy.py:77
msgid "Enabled"
msgstr "Omogočeno"

#: redbot/cogs/audio/core/events/dpy.py:77
msgid "Disabled"
msgstr "Onemogočeno"

#: redbot/cogs/audio/core/events/dpy.py:124
msgid "No DJ role found. Disabling DJ mode."
msgstr "DJ rola ne obstaja. Izklapljam DJ način."

#: redbot/cogs/audio/core/events/dpy.py:134
msgid "`{user_input}` is not a valid value for `{command}`"
msgstr "Vrednost `{user_input}` ni veljavna za `{command}`"

#: redbot/cogs/audio/core/events/dpy.py:142
msgid "Unable To Parse Argument"
msgstr "Argumenta ni mogoče razčleniti"

#: redbot/cogs/audio/core/events/dpy.py:154
#: redbot/cogs/audio/core/events/dpy.py:163
msgid "Invalid Argument"
msgstr "Neveljaven argument"

#: redbot/cogs/audio/core/events/dpy.py:155
msgid "The argument you gave for `{}` is not valid: I was expecting a `{}`."
msgstr "Podani argument za `{}` ni veljaven: Pričakoval sem `{}`."

#: redbot/cogs/audio/core/events/dpy.py:175
msgid "Invalid Environment"
msgstr "Neveljavno okolje"

#: redbot/cogs/audio/core/events/dpy.py:176
msgid "Connection to Lavalink has been lost."
msgstr "Poveza do Lavalink je bila prekinjena."

#: redbot/cogs/audio/core/events/dpy.py:184
msgid "No Player Available"
msgstr "Na voljo ni noben predvajalnik"

#: redbot/cogs/audio/core/events/dpy.py:185
msgid "The bot is not connected to a voice channel."
msgstr "Bot ni povezan do zvočnega kanala."

#: redbot/cogs/audio/core/events/dpy.py:193
#: redbot/cogs/audio/core/events/lavalink.py:162
msgid "Unable to Get Track"
msgstr "Skladbe ni mogoče dobiti"

#: redbot/cogs/audio/core/events/dpy.py:194
#: redbot/cogs/audio/core/events/lavalink.py:163
msgid "I'm unable to get a track from Lavalink at the moment, try again in a few minutes."
msgstr "Trenutno ne morem dobiti skladb od Lavalinka, poskusite znova čez nekaj minut."

#: redbot/cogs/audio/core/events/dpy.py:205
msgid "There was an issue communicating with Discord."
msgstr "Prišlo je do težave pri komunikaciji z Discordom."

#: redbot/cogs/audio/core/events/dpy.py:206
msgid "This error has been reported to the bot owner."
msgstr "Napaka je bila posredovana lastniku bota."

#: redbot/cogs/audio/core/events/lavalink.py:154
msgid "Couldn't get a valid track."
msgstr "Ni bilo mogoče dobiti veljavne skladbe."

#: redbot/cogs/audio/core/events/lavalink.py:189
msgid "Now Playing"
msgstr "Trenutno se predvaja"

#: redbot/cogs/audio/core/events/lavalink.py:191
msgid "Track length: {length} | Requested by: {user}"
msgstr "Dolžina skladbe: {length} | Zahteval to je {user}"

#: redbot/cogs/audio/core/events/lavalink.py:212
msgid "Queue ended."
msgstr "Čakalna vrsta se je končala."

#: redbot/cogs/audio/core/events/lavalink.py:264
msgid "Multiple Errors Detected"
msgstr "Zaznanih je več napak"

#: redbot/cogs/audio/core/events/lavalink.py:265
msgid "Closing the audio player due to multiple errors being detected. If this persists, please inform the bot owner as the Audio cog may be temporally unavailable."
msgstr "Zapiranje predvajalnika zvoka zaradi zaznanih več napak. Če se to ponavlja, prosimo obvesti lastnika bota saj je možno da je Audio modul trenutno nedostopen."

#: redbot/cogs/audio/core/events/lavalink.py:279
msgid "Track Stuck"
msgstr "Skladbe prekinjena"

#: redbot/cogs/audio/core/events/lavalink.py:280
msgid "Playback of the song has stopped due to an unexpected error.\n"
"{error}"
msgstr ""

#: redbot/cogs/audio/core/events/lavalink.py:286
msgid "Track Error"
msgstr "Napaka skladbe"

